lib/host.py

```diff
@@ -397,14 +397,22 @@ def disks(self):
     def disk_is_available(self, disk):
         return len(self.ssh(['lsblk', '-n', '-o', 'MOUNTPOINT', '/dev/' + disk]).strip()) == 0
 
-    def available_disks(self):
+    def available_disks(self, blocksize="512"):
```

Comment:
a size being a string looks weird to me.
I'd make an int and use `str(blocksize)`, or even better `int(sec_size)`, when comparing to the return of `lsblk`